packages/patternfly-4/react-table/src/components/Table/index.d.ts

```diff
@@ -19,5 +19,5 @@ export { default as TableHeader, HeaderProps } from './Header';
 export { default as TableBody, TableBodyProps } from './Body';
 export { default as RowWrapper, RowWrapperRow, RowWrapperProps } from './RowWrapper';
 export { default as ExpandableRowContent } from './ExpandableRowContent';
-export { sortable, headerCol, cellWidth, ISortable, expandable, isRowExpanded } from './utils';
+export { sortable, headerCol, cellWidth, ISortable, expandable, isRowExpanded, compoundExpand } from './utils';
```

Comment:
This seems close to `export * from './utils';`

Either way, the TS conversion in #2360 will eventually resolve this too.

Reply:
+1... i am planning to circle back and ensure we can do `export *` on all exports, and that they are converted from default exports to named exports... will probably attempt this towards the end to be certain. I've gone ahead and rebased this change for now though!